benchmark/benchmarks/krausest/vite.config.mts

```diff
@@ -2,15 +2,44 @@ import fs from 'node:fs';
 
 import { precompile } from '@glimmer/compiler';
 import { defineConfig, type Plugin } from 'vite';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+const self = import.meta.url;
+
+const packagesPath = path.resolve(path.dirname(fileURLToPath(self)), '..', '..', './../packages');
+
+const packagePath = (name: string) => {
+  return path.join(packagesPath, name, 'dist/prod/index.js');
+};
 
 export default defineConfig({
   plugins: [benchmark()],
+  resolve: {
+    alias: {
+      '@glimmer-workspace/benchmark-env': '@glimmer-workspace/benchmark-env/index.ts',
+      '@glimmer/debug': packagePath('@glimmer/debug'),
+    },
+  },
 });
 
 function benchmark(): Plugin {
   return {
+    enforce: 'pre',
     name: '@glimmer/benchmark',
+    resolveId(id) {
+      if (id === '@glimmer/env') {
+        return '\0@glimmer/env';
+      } else if (id === '@glimmer/local-debug-flags') {
+        return '\0@glimmer/local-debug-flags';
+      }
+    },
     load(id) {
```

Comment:
is this needed? the build / package.json#exports should have already handled this :sweat_smile:

Reply:
oh I see it was pre-existing from here: https://github.com/glimmerjs/glimmer-vm/pull/1532/files#diff-860eda2adf31f087997e1c92eab487cb90113b3f303653d36adabd42213d5c42L26 

We can probably evaluate removing / cleaning it up in a separate PR then.